server/src/services/IngredientsConversion.js

```diff
@@ -0,0 +1,107 @@
+class IngredientsConversion {
```

Comment:
this name is fine, but `IngredientMeasurementConversion` or `IngredientMeasurementConverter` could be even better!